domehunter/dome_control.py

```diff
@@ -245,7 +246,11 @@ def __init__(self,
     @property
     def dome_az(self):
         """ """
-        return self._ticks_to_az(self._encoder_count)
+        if self._dome_az is None:
+            print("Cannot return Azimuth as Dome is not yet calibrated.\
+                   Run calibration loop")
```

Comment:
Possibly misleading statement. Dome azimuth will be unknown if the dome has not been calibrated (should be a rare occurrence) or if it has just not been homed (more frequent).

Reply:
The `_dome_az` should only be at `None` before calibration. At home it should read the proper azimuth.

Reply:
I think my point was that shouldn't be the case. We shouldn't be recalibrating every time the system starts (that should only be needed if the encoder hardware has changed, i.e. hardly ever), normally the Dome class should be initialised with valid `degrees_per_tick` and `home_azimuth` values so that calibration is unnecessary. The dome azimuth will be still be unknown (and should be `None`) until the dome has found home, however.

So, the correct response to `dome_azimuth` being `None` would normally be to call `.find_home()`. `.calibrate_dome_encoder_counts()` would only be required if `degrees_per_tick` is also unknown. Unfortunately `degrees_per_tick` is currently being given a default value of 1.0 so that it will never be `None`, and there is know way to know whether the dome is in fact calibrated.